src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -156,6 +168,9 @@ public class Course extends DomainObject {
     @Column(name = "registration_confirmation_message")
     private String registrationConfirmationMessage;
 
+    @Column(name = "disenrollment_enabled")
+    private Boolean disenrollmentEnabled;
```

Comment:
```suggestion
    private boolean disenrollmentEnabled = false;
```
Early suggestion: Please make this a non-nullable data type and also set a default value in the database definition.

Booleans with `null` are tricky since there are multiple ways to handle that as `true` or `false` depending on context, and all comparisons have to be made using `Booleans.TRUE.equals(…)` or similar.

Reply:
Thx for the suggestion. I changed it immediately: [commit](https://github.com/ls1intum/Artemis/pull/6577/commits/4d5af9382f28f2bb615850321a7b1b6b72b38d96)